tests/integration/run-secure-handlebars-spec.js

```diff
@@ -90,7 +90,24 @@ describe("SecureHandlebars: compilation tests", function() {
             'url': 'javascript:alert(1)'
         });
 
-        expect(output).to.be.equal('<!--hello--><a href=x-javascript:alert(1) style="background:url(##javascript:alert\\28 1\\29 )" id="12`&quot;\'3" id=\'12`"&#39;3\'>hello</a>');
+        expect(output).to.be.equal('<!--hello--><a href=x-javascript:alert(1) style="background:url(##javascript:alert%281%29)" id="12`&quot;\'3" id=\'12`"&#39;3\'>hello</a>');
+    });
+});
+
+describe("SecureHandlebars: preprocess tests", function() {
+    it('smoke template', function(){
+        var html = '<!--{{hello}}--><a href={{url}} style="background:url({{url}})" id="{{id}}" id=\'{{id}}\'>{{hello}}</a>';
+        var data = {
+            'id': '12`"\'3',
+            'hello': 'hello',
+            'url': 'javascript:alert(1)'
+        };
+
+        var preprocessed = Handlebars.preprocess(html);
```

Comment:
not really new. it's been exposed quite some time, and is also tested in unit tests. this is to only enhance the coverage of integration test.